microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,139 +20,175 @@ ms.date: 05/04/2021
 
-   ![The Open File dialog.](../media/resources-command-menu-empty.msft.png)
+1. When DevTools has focus, press `Ctrl`+ `P` (Windows, Linux) or `Command`+`P` (macOS). Or, you can click the (![Customize icon](../media/customize-devtools-icon-light-theme.png)) menu button in DevTools and then select **Open file**.  The Command Menu opens, displaying the **Open File** list:
 
-1. Select the file from the dropdown list, or start typing the filename and press `Enter` once the correct file is highlighted in the autocomplete box.
+   ![The Command Menu, displaying the Open File list](../media/resources-command-menu-empty.msft.png)
 
-   ![Type a filename in the Open File dialog.](../media/resources-command-menu-file-search.msft.png)
+1.  If there's a greater-than (>) prompt, press `Backspace` to get to the **Open file** prompt.
+
+1. Start typing the filename, and then press `Enter` when the correct file is highlighted in the autocomplete box, or select the file from the dropdown list:
+
+   Type filename:
+
+   ![Typing part of a filename in the Open File list of the Command Menu](../media/resources-command-menu-file-search.msft.png)
 
 
 <!-- ====================================================================== -->
-## Open resources in the Network tool
+## Browse resource files in the Network tool
 
-See [Inspect the details of a resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
-<!--why?-->
+In the **Network** tool, you can inspect the resource files that make up the current webpage, such as `.html`, `.css`, `.js`, and image files.  To learn how to get the details for a specific resource, see [Inspect the details of the resource](../network/index.md#inspect-the-details-of-the-resource) in _Inspect network activity_.
 
-<!-- need lead-in -->
+When you go to the **Network** tool and then refresh the page, all the webpage's resource files are listed in the **Network** tool, one row per resource file:
 
-![Inspect a resource in the Network tool.](../media/resources-network-response.msft.png)
+![Webpage resource files listed in the Network Log, one resource file per row](../media/resources-network-resources.msft.png)
+
+When **All** is selected as the resources filter, all the retrieved resource files for the current webpage are listed, in the lower left:
 
-### Reveal resources in the Network tool from other panels
+![Inspect a resource in the Network tool](../media/resources-network-response.msft.png)
 
-The next section, [Browse resources in the Network panel](#browse-resources-in-the-network-panel), shows how to view resources from various parts of the DevTools UI.  To inspect a resource in the **Network** tool, right-click the resource and select **Reveal in Network panel**.
+The above screenshot shows `getstarted.html`, though a folder is selected in the lower left.
```

Comment:
```suggestion
The above screenshot shows `index.html`, for the `network-tutorial` path or folder that's selected in the lower left.  The screenshot shows the [Inspect Network Activity Demo](https://microsoftedge.github.io/Demos/network-tutorial/).  The source files for that page are at [MicrosoftEdge / Demos > network-tutorial](https://github.com/MicrosoftEdge/Demos/tree/main/network-tutorial).
```

Reply:
@dan-wesley - did you mean to apply this Suggestion?  Appears it wasn't applied though marked Resolved.  This fixes an error in the text about which .html file is shown in screenshot.

Reply:
just added locally - I'm finding that 'Add suggestion to batch' and 'Commit suggestion' don't also work the way I think they should.  And in some cases when I pulled down the changes I got dups in my local copy